content/guides/tinacms/separate-content-repo/guide.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Separate Content Repo
+last_edited: '2023-01-24T10:00:00.000Z'
+---
+
+Tina supports sourcing content from a separate Git repo. With this approach you can continue to define your
+Tina config in your website repo, but documents will be saved elsewhere.
+
+## Why?
+
+You might want to do this for a variety of reasons, such as:
+
+- To decouple your website code from your content files so your commit history isn't a mix of content updates and code updates
+- To avoid having to give Tina Cloud write access to your website's code (only give access to your content repo)
+
+## Walkthrough
+
+This guide will start by having you clone the "website repo" locally. From there, you'll be creating the
+"content repo" so you can start by cloning the project [here](https://github.com/tinacms/separate-website-repo):
+
+```sh
+git clone git@github.com:tinacms/separate-website-repo.git
+```
+
+The workflow requirements are:
+
+- You must have _both_ the "content repo" and the "website repo" checked out onto your local machine.
+- You must provide the location of your content repo in your Tina config (more about that below).
+- When making changes, you'll need to ensure you've pushed _both_ repos to Github, with the "content repo"
+  changes first
+
+## Create the content repository
+
+Using a separate content repo works by defining your config in the repo you'll use for _editing_ with the CMS (which is this repo). We'll call that the "website repo".
+To set up the "content repo", we'll start with a simple `.mdx` file:
```

Comment:
This paragraph feels a tad weird because we're talking about the website repo, then switch to the instruction of setting up the website repo

Reply:
What do you suggest instead?

Reply:
> Using a separate content repo works by defining your config in the repo you'll use for editing with the CMS (which is this repo). We'll call that the website repo.

^ I'd probably suggest moving that stuff out of the "Create the content repository" heading. Maybe there should be a separate "Creating the website repository" section?